src/libs/highlevel/README.md

```diff
@@ -30,15 +30,13 @@ if (elektra == NULL)
 }
 
 int myint = elektraGetLong (elektra, "myint");
+printf ("got long %d\n", myint);
 
-if (myint < 10)
+elektraSetBoolean (elektra, "mybool", true, &error);
```

Comment:
The idea of this change is that we should not demonstrate dependencies between configs.